general/solomiia_tymoshchuk/Task7.java

```diff
@@ -0,0 +1,68 @@
+package general.solomiia_tymoshchuk;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Write a program that prints all odd numbers from N to 1.
+ * N should be read from the input.
+ * Every number should be printed on the new line.
+ * Create two versions of the program: one should use the for loop,
+ * second - the while loop.
+ **/
+public class Task7 {
+    Scanner sc = new Scanner(System.in);
+    int number = sc.nextInt();
+    List<Integer> newarr = new ArrayList<Integer>();
+
+    void oddNumbers() {
+        for (int k = getNumber(); k >= 0; k--) {
```

Comment:
since number is an instance variable, you can access it directly from the instance method